xcpmd/src/vm-actions-module.c

```diff
@@ -707,9 +767,11 @@ void shutdown_vpnvm_dependencies_of_vm_by_name (struct arg_node * args) {
 void shutdown_vpnvm_dependencies_of_vm_by_uuid (struct arg_node * args) {
```

Comment:
I guess it's fine.
In a later PR, maybe the indentation could be fixed and `shutdown_dependencies_of_vm()` moved out of the way.